debian/patches/pop_upgrade.patch

```diff
@@ -0,0 +1,283 @@
++      POP_UPGRADE,
++      (PopUpgradeWidgetReadyCallback) pop_upgrade_callback_ready,
++      NULL
++    );
++  }
++
++  if (g_atomic_int_get (&POP_UPGRADE_STATUS) == 0) {
++    pop_upgrade_widget_scan (POP_UPGRADE);
++  }
++
++  GtkWidget *upgrade_widget = pop_upgrade_widget_container (POP_UPGRADE);
++  gtk_widget_set_margin_top (upgrade_widget, 12);
++  gtk_container_add (GTK_CONTAINER (priv->column_box), upgrade_widget);
++  pop_upgrade_widget_callback_error (POP_UPGRADE, (PopUpgradeWidgetErrorCallback) pop_upgrade_callback_error, priv);
++
++  // End of Pop!_OS Upgrade Integration
++
+   info_overview_panel_setup_overview (self);
+   info_overview_panel_setup_virt (self);
+ }
+Index: gnome-control-center/panels/info/cc-info-overview-panel.ui
+===================================================================
+--- gnome-control-center.orig/panels/info/cc-info-overview-panel.ui
++++ gnome-control-center/panels/info/cc-info-overview-panel.ui
+@@ -10,6 +10,20 @@
+         <property name="can-focus">False</property>
+         <property name="hscrollbar-policy">never</property>
+         <child>
++          <!-- Pop!_OS wrap view in scrolled window -->
++          <object class="GtkOverlay" id="overlay">
++          <child>
++          <object class="GtkScrolledWindow">
++          <property name="visible">True</property>
++          <property name="valign">fill</property>
++          <property name="halign">fill</property>
++          <property name="hscrollbar-policy">never</property>
++          <property name="vscrollbar-policy">automatic</property>
++          <property name="propagate-natural-height">True</property>
++          <property name="shadow-type">in</property>
++          <property name="margin">0</property>
++          <child>
++          <!-- -->
+           <object class="HdyColumn">
+             <property name="visible">True</property>
+             <property name="maximum_width">600</property>
+@@ -19,7 +33,7 @@
+             <property name="margin_start">12</property>
+             <property name="margin_end">12</property>
+             <child>
+-              <object class="GtkBox">
++              <object class="GtkBox" id="column_box">
+                 <property name="visible">True</property>
+                 <property name="can_focus">False</property>
+                 <property name="valign">center</property>
+@@ -359,5 +373,6 @@
+         </child>
+       </object>
+     </child>
++    </object></child></object></child>
+   </template>
+ </interface>
+Index: gnome-control-center/shell/cc-application.c
+===================================================================
+--- gnome-control-center.orig/shell/cc-application.c
++++ gnome-control-center/shell/cc-application.c
+@@ -197,6 +197,10 @@ cc_application_quit (GSimpleAction *simp
+                      GVariant      *parameter,
+                      gpointer       user_data)
+ {
++  if (g_atomic_int_get (&POP_UPGRADE_STATUS) == 1) {
```

Comment:
This doesn't make a difference - when this function returns the application will terminate. You need to edit the window like the installer does to make it not closable